src/main/java/com/sygic/travel/sdk/tours/service/ToursService.kt

```diff
@@ -1,18 +1,36 @@
 package com.sygic.travel.sdk.tours.service
 
 import com.sygic.travel.sdk.common.api.SygicTravelApiClient
-import com.sygic.travel.sdk.places.model.query.ToursQuery
+import com.sygic.travel.sdk.places.model.query.ToursGetYourGuideQuery
+import com.sygic.travel.sdk.places.model.query.ToursViatorQuery
 import com.sygic.travel.sdk.tours.model.Tour
 
 internal class ToursService(
 	private val apiClient: SygicTravelApiClient
 ) {
-	fun getTours(toursQuery: ToursQuery): List<Tour>? {
-		val request = apiClient.getTours(
-			destinationId = toursQuery.destinationId,
-			page = toursQuery.page,
-			sortBy = toursQuery.sortBy?.string,
-			sortDirection = toursQuery.sortDirection?.string
+	fun getToursViator(toursViatorQuery: ToursViatorQuery): List<Tour>? {
+		val request = apiClient.getToursViator(
+			parentPlaceId = toursViatorQuery.parentPlaceId,
+			page = toursViatorQuery.page,
+			sortBy = toursViatorQuery.sortBy?.string,
+			sortDirection = toursViatorQuery.sortDirection?.string
+		)
+		return request.execute().body()?.data?.getTours()
```

Comment:
I'm thinking that we should throw in case of error state, not return null, what do you think?

Reply:
I just mean !!.data!!.getTrous().

Reply:
So we crash an app, if API returns error response.

Reply:
No, we throw NullPointer, that's not exactly a crash :) I would modified it that way. The same way you do not check for Dns resolution exceptions etc... the client probably should try/catch the SDK calls.

Reply:
We don't do this in any of the SDK's service classes 🤔